### PR TITLE
added possibilty to change units between month and year

### PR DESCRIPTION
**Pull request by WMeck**

### CODE REVIEW

**Comment by WMeck** [comment]
I have added on new branch possibility to change tenary untis between Months and Years

File: src/components/EMI_Components/FormInput.jsx [hunk 1]
```diff
@@ -39,7 +58,20 @@ const FormInput = ({
...
+          {!visible && <InputGroupText>{symbol}</InputGroupText> }
+          { visible &&
+          
+          <ButtonDropdown isOpen={isOpen} toggle={toggle}>                  
+          <DropdownToggle caret>
+              {unit}
+            </DropdownToggle>
+            <DropdownMenu>
+             {units.map((val) => {
+              return <DropdownItem onClick={() => handleUnitChange(val)}>{val}</DropdownItem>
```
Comment (ivangarcilazo) [inline src/components/EMI_Components/FormInput.jsx:70]: image

It works fine and looks good. Just add the key to avoid this problem.

**Review by ivangarcilazo — COMMENTED** [review]
Fix that warning, it looks good :D

**Comment by WMeck** [comment]
Added key value in new commit

**Review by ivangarcilazo — APPROVED** [review]

**Pull request merged**